src/cli.nim

```diff
@@ -141,7 +155,7 @@ proc processCmdLine*: Conf =
       of optVerbosity.short, optVerbosity.long:
         showErrorForMissingVal(kind, key, val)
         result.verbosity = parseVerbosity(kind, key, val)
-      of optProbSpecsDir.short, optProbSpecsDir.long:
+      of optProbSpecsDir.short, optProbSpecsDir.long.toLowerAscii():
```

Comment:
After reading your reasoning I understand the reason behind this, but I found it slightly hard to understand. Let's look at the best way to do this later.

Reply:
To others: it was just a temporary workaround until https://github.com/exercism/canonical-data-syncer/pull/86 was merged.

The current implementation on `master` is:
https://github.com/exercism/canonical-data-syncer/blob/ae310285943fb9c404f522ad27a78fc592f629e5/src/cli.nim#L185-L188